src/Forms/Compare.cs

```diff
@@ -314,7 +319,11 @@ private void CompareSubtitles()
                     else
                     {
                         int columnsAlike = GetColumnsEqualExceptNumber(p1, p2);
-                        if (columnsAlike >= 0 && columnsAlike < 4)

```

Comment:
Ah, this is necessary in order to make the coloring work... one of the bugs I fixed.